crates/exex/exex/src/wal/entry.rs

```diff
@@ -0,0 +1,25 @@
+use reth_exex_types::ExExNotification;
+use serde::{Deserialize, Serialize};
+
+/// A single entry in the WAL.
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub struct WalEntry {
+    pub(crate) target: NotificationCommitTarget,
+    pub(crate) notification: ExExNotification,
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub(crate) enum NotificationCommitTarget {
+    Commit,
+    Canonicalize,
+}
```

Comment:
dont fully understand why we need to track this?
is this so we know whether we modified this?